lib/evaluation/handlers/uriEvaluationHandler.ts

```diff
@@ -89,7 +103,7 @@ export class UriEvaluationHandler extends AbstractEvaluationHandler {
     const result: HandlerCheckResult = this.createResult(idIdx, vcIdx);
     result.status = Status.INFO;
     result.message = PEMessages.URI_EVALUATION_PASSED;
-    result.payload = { presentationDefinitionUris: verifiableCredentialUris, inputDescriptorsUris };
+    result.payload = { verifiableCredentialUris, inputDescriptorsUris };
```

Comment:
What is the reason for the verifiableCredentialUris in the payload? I think it is pretty confusing, because it is merging two separate distinct VC properties into a single array

Reply:
this just a reference to show the user (in case of a error) which values have been compared to which

Reply:
Then we really should rename it into something more meaningful, like contextAndSchemaUris or something

Reply:
I'm separating these two sources. one will be `vcContextUri` and the other `vcCredentialSchema`